### PR TITLE
Ensure no null obj is passed when tags are null + update FATS

### LINKED ISSUE #25152: Request Timing metrics not showing up with `mpMetrics-5.0` (when used with `requestTiming-1.0` feature.
**Describe the bug**  
Request Timing metrics not showing up with `mpMetrics-5.0` (when used with `requestTiming-1.0` feature.

**Steps to Reproduce**   + **Expected behavior**  
When enabled with `requestTiming-1.0` and `mpMetrics-5.0` we expect to see request timing metrics.

**Diagnostic information:**  
 - OpenLiberty Version: 23.0.0.3
 - Affected feature(s) requestTiming-1.0` and `mpMetrics-5.0`
 - Java Version: N/A

### PR DESCRIPTION
**Pull request by Channyboy**
fixes #25152 
#build

### CODE REVIEW

File: dev/com.ibm.ws.request.timing.monitor_fat/fat/src/com/ibm/ws/request/timing/monitor/fat/FATSuite.java [hunk 1]
```diff
@@ -24,10 +24,12 @@
...
                 RequestTimingEventTest.class,
-                RequestTimingMetricsTest.class })
+                RequestTimingMetricsTest.class
+})
+
 public class FATSuite {
 
     @ClassRule
-    public static RepeatTests r = RepeatTests.withoutModification().andWith(new FeatureReplacementAction("mpMetrics-3.0", "mpMetrics-2.3").withID("MPM23")).andWith(new FeatureReplacementAction("mpMetrics-2.3", "mpMetrics-2.2").withID("MPM22")).andWith(new FeatureReplacementAction("mpMetrics-2.2", "mpMetrics-2.0").withID("MPM20")).andWith(FeatureReplacementAction.EE9_FEATURES().removeFeature("mpMetrics-2.0")).andWith(FeatureReplacementAction.EE10_FEATURES().removeFeature("mpMetrics-2.0"));
+    public static RepeatTests r = RepeatTests.withoutModification().andWith(FeatureReplacementAction.EE9_FEATURES()).andWith(FeatureReplacementAction.EE10_FEATURES());
```
Comment (Channyboy) [inline dev/com.ibm.ws.request.timing.monitor_fat/fat/src/com/ibm/ws/request/timing/monitor/fat/FATSuite.java:33]: This repeats with EE9 and EE8 for the `RequestTimingMbeanTest` and `RequestTimingEventTest` tests. The `RequestTimingMetricsTest` will run during the without modifications (and do its own class repeats w.r.t to the MP repeat actions).

File: dev/com.ibm.ws.request.timing.monitor_fat/fat/src/com/ibm/ws/request/timing/monitor/fat/RequestTimingMetricsTest.java [hunk 2]
```diff
@@ -55,6 +58,16 @@
 @SkipForRepeat({ JakartaEE9Action.ID, JakartaEE10Action.ID })
```
Comment (Channyboy) [inline dev/com.ibm.ws.request.timing.monitor_fat/fat/src/com/ibm/ws/request/timing/monitor/fat/RequestTimingMetricsTest.java:58]: Skips the EE9 and EE10 repeats ( they do not replace MP actions).

File: dev/com.ibm.ws.request.timing.monitor_fat/fat/src/com/ibm/ws/request/timing/monitor/fat/RequestTimingMetricsTest.java [hunk 3]
```diff
@@ -84,6 +103,28 @@ public static void setUp() throws Exception {
 
         Log.info(c, "setUp", logMsg);
         Assert.assertNotNull("No CWWKF0011I was found.", logMsg);
+
+        resolveRequestTimingMetricString();
+
+    }
+
+    private static void resolveRequestTimingMetricString() throws Exception {
```
Comment (Channyboy) [inline dev/com.ibm.ws.request.timing.monitor_fat/fat/src/com/ibm/ws/request/timing/monitor/fat/RequestTimingMetricsTest.java:111]: Main change to the test. Need to resolve the vendor /metrics url and the format of the metric names. Depending on mpMetrics feature level

File: dev/io.openliberty.microprofile.metrics.5.0.monitor.internal/src/io/openliberty/microprofile/metrics/internal/monitor/internal/MonitorMetrics.java [hunk 4]
```diff
@@ -113,8 +113,12 @@ public void createMetrics(SharedMetricRegistries sharedMetricRegistry, String[][
 
                         SRMetricRegistryAdapter srMetricRegistry = new SRMetricRegistryAdapter(cast);
 
-                        srMetricRegistry.functionCounter(metadata, mc, x -> (x.getCount() * conversionFactor),
-                                metricTag);
+                        if (metricTag == null) {
```
Comment (Channyboy) [inline dev/io.openliberty.microprofile.metrics.5.0.monitor.internal/src/io/openliberty/microprofile/metrics/internal/monitor/internal/MonitorMetrics.java:116]: Passing null tags will cause a NPE in SR. Easier/Quicker to fix here.

**Review by fmhwong — APPROVED** [review]
LGTM

**Pull request merged**